index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <h1>Hello Microverse</h1>
```

Comment:
- [ Optional ] As stated in the project requirement kindly i recommend you  to use class for your header("Hello Microverse") to style with your favorite color  just like this `h1 class="heading"`.